src/Components/App/App.js

```diff
@@ -8,56 +8,68 @@ const logo = require('../../Images/cherry-tomato.png')
 
 class App extends Component {
   constructor() {
-    super() 
+    super()
     this.state = {
       movies: [],
       singleMovie: {},
       movieID: "",
-      rating: null,
-      error: "", 
-    } 
+      rating: 0,
+      error: "",
+      filteredMovies: [],
+      loading: false
```

Comment:
I love this loading property. It's very simple and useful to us to to determine if the site is loading